jerry-core/ecma/base/ecma-helpers.c

```diff
@@ -1164,30 +1123,31 @@ ecma_set_named_accessor_property_setter (ecma_object_t *object_p, /**< the prope
  *         false - otherwise.
  */
 inline bool __attr_always_inline___
-ecma_is_property_writable (ecma_property_t *prop_p) /**< property */
+ecma_is_property_writable (ecma_property_t property) /**< property */
 {
-  JERRY_ASSERT (ECMA_PROPERTY_GET_TYPE (prop_p) == ECMA_PROPERTY_TYPE_NAMEDDATA);
+  JERRY_ASSERT (ECMA_PROPERTY_GET_TYPE (property) == ECMA_PROPERTY_TYPE_NAMEDDATA
+                || ECMA_PROPERTY_GET_TYPE (property) == ECMA_PROPERTY_TYPE_VIRTUAL);
 
-  return (prop_p->type_and_flags & ECMA_PROPERTY_FLAG_WRITABLE) != 0;
+  return (property & ECMA_PROPERTY_FLAG_WRITABLE) != 0;
 } /* ecma_is_property_writable */
 
 /**
  * Set property's 'Writable' attribute value
  */
 void
-ecma_set_property_writable_attr (ecma_property_t *prop_p, /**< property */
+ecma_set_property_writable_attr (ecma_property_t *property_p, /**< property */
```

Comment:
`/**< [in,out] property */`
It would be nice to tag the similar function parameters that you touched in this PR.